domain/tshirt.js

```diff
@@ -10,19 +10,26 @@ function invalidData(body) {
   return _.isUndefined(body.size) || _.isUndefined(body.model);
 }
 
+tshirt.NoTShirt = function () {
```

Comment:
I capitalized to indicate that this is the NullObject of Tshirt rather than a method of a tshirt. I am not sure what is the convention if js for that. what do you think?